templates/openstack/index.html

```diff
@@ -0,0 +1,131 @@
+        <p class="p-card__content"><a href="/download/cloud/conjure-up" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Internal Link', 'eventAction' : 'Cloud openstack', 'eventLabel' : 'Try conjure-up', 'eventValue' : undefined });">Install OpenStack manually&nbsp;&rsaquo;</a></p>
+      </div>
+    </div>
+  </div>
+</section>
+
+<section class="p-strip is-bordered">
+  {% include "cloud/shared/_openstack_clients.html" %}
+</section>
+
+<section class="p-strip--accent is-deep">
+  <div class="row">
+    <div class="u-equal-height">
+      <div class="col-7 ">
+        <h2>BootStack &mdash; fully managed OpenStack on premises</h2>
+        <p>BootStack a private OpenStack cloud managed remotely by Canonical.  This is the fastest way to get to production. It runs on your hardware, in the datacenter of your choice with our experts responsible for deployment and 24/7 support.</p>
+        <ul class="p-list">
+          <li class="p-list__item is-ticked">Built to your specifications</li>
+          <li class="p-list__item is-ticked">Your choice of Ubuntu- certified server and network hardware</li>
+          <li class="p-list__item is-ticked">Monitor every machine with your existing dashboards</li>
+          <li class="p-list__item is-ticked">Guaranteed availability backed by a clear SLA</li>
+          <li class="p-list__item is-ticked">Transfer operations anytime you want</li>
+        </ul>
+        <p><a href="/cloud/managed-cloud" class="p-link--inverted">Learn more about BootStack&nbsp;&rsaquo;</a></p>
+      </div>
+      <div class="col-5 u-vertically-center u-align--center u-hidden--small">
+        <img src="{{ ASSET_SERVER_URL }}466046c7-illustration-cloud-bootstack.svg?w=320" alt="" class="u-hidden—small" width="320" />
+      </div>
+    </div>
+  </div>
+</section>
+
+<section class="p-strip is-deep is-bordered">
+  <div class="row">
+    <div class="col-7">
+      <h2>We build you a production OpenStack in 3 weeks</h2>
+    </div>
+  </div>
+  <div class="row">
+    <div class="col-7">
+      <p>With our Foundation Cloud Build service you can get a full production OpenStack cloud up and running in 3 weeks using the reference architecture. With model driven operations and upgrades to future OpenStack releases guaranteed, the OpenStack cloud from Canonical is the fast track to efficient cloud success. Optional consulting available to tailor the architecture to your specific requirements.</p>
+      <ul class="p-list is-split">
+        <li class="p-list__item is-ticked">SLAs and support services</li>
+        <li class="p-list__item is-ticked">On-site engineering and support</li>
+        <li class="p-list__item is-ticked">Reference architecture available</li>
+        <li class="p-list__item is-ticked">OpenStack feature development as needed</li>
+        <li class="p-list__item is-ticked">Customized architecture service placement</li>
```

Comment:
Wrong spelling of customized, I've corrected the doc

Reply:
Thanks